docs/EAP2.md

```diff
@@ -0,0 +1,5 @@
+# Declarative Gradle - November 2024 Update
+
+!!! info
```

Comment:
We can embed the post later if needed.
I have embeddable version for the newsletter, and that code can be ported

Reply:
Just like EAP.md, this is a redirect, see mkdocs.yml